openwisp_controller/pki/migrations/0010_common_name_organization_unique.py

```diff
@@ -0,0 +1,78 @@
+# Generated by Django 3.1.12 on 2021-07-06 12:22
+
+import shortuuid
+from django.db import migrations, models
+
+from . import get_swapped_model
+
+
+def make_ca_common_name_unique(apps, schema_editor):
+    Ca = get_swapped_model(apps, 'django_x509', 'Ca')
+    updated_cas = []
+    for ca in Ca.objects.iterator():
+        qs = Ca.objects.filter(
+            common_name=ca.common_name, organization_id=ca.organization_id
+        ).exclude(pk=ca.pk)
+        for ca_ in qs.iterator():
+            common_name = ca_.common_name
+            unique_slug = shortuuid.ShortUUID().random(length=8)
+            common_name = f'{common_name}-{unique_slug}'
+            ca_.common_name = common_name
+            updated_cas.append(ca_)
+            if len(updated_cas) > 1000:
+                Ca.objects.bulk_update(updated_cas, ['common_name'])
+                updated_cas = []
+    if updated_cas:
+        Ca.objects.bulk_update(updated_cas, ['common_name'])
+
+
+def make_cert_common_name_unique(apps, schema_editor):
+    Cert = get_swapped_model(apps, 'django_x509', 'Cert')
+    VpnClient = get_swapped_model(apps, 'config', 'VpnClient')
+    for cert in Cert.objects.iterator():
+        qs = Cert.objects.filter(
+            common_name=cert.common_name, organization_id=cert.organization_id
+        ).exclude(pk=cert.pk)
+        for cert_ in qs.iterator():
```

Comment:
what about using `dupe_cn_cert` instead of `cert_`? Similar suggestion for `ca_`above.